cmd/operator-opamp-bridge/Dockerfile

```diff
@@ -16,4 +16,7 @@ COPY --from=certificates /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-ce
 # Copy binary built on the host
 COPY bin/opampbridge_${TARGETARCH} ./main
 
+# "nonroot"
+USER 65532:65532
```

Comment:
Note I used 65532 as it's the distroless nonroot user, which the main Dockerfile uses.